test/scripts/teal-split-test.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+date '+teal-split-test start %Y%m%d_%H%M%S'
+
+set -e
+set -x
+
+TEMPDIR=$(mktemp -d)
+trap "rm -rf $TEMPDIR" 0
+
+NETDIR=${TEMPDIR}/net
+
+if [ ! -z $BINDIR ]; then
+    export PATH=${BINDIR}:${PATH}
+fi
+
+goal network create -r ${NETDIR} -n tbd -t ${GOPATH}/src/github.com/algorand/go-algorand/test/testdata/nettemplates/TwoNodes50EachFuture.json
+
+goal network start -r ${NETDIR}
+
+# replaces prior trap0
+trap "goal network stop -r ${NETDIR}; rm -rf ${TEMPDIR}" 0
+
+export ALGORAND_DATA=${NETDIR}/Node
+
+ACCOUNT=$(goal account list|awk '{ print $3 }')
+
+ACCOUNTB=$(goal account new|awk '{ print $6 }')
+ACCOUNTC=$(goal account new|awk '{ print $6 }')
```

Comment:
Just want to confirm--you intended this to be the same as `$ACCOUNTB`, right?

Reply:
same action, but again, so as to make two accounts B and C